internal/repository/rsl.go

```diff
@@ -187,3 +188,31 @@ func (r *Repository) isDuplicateEntry(refName string, targetID plumbing.Hash) (b
 
 	return latestUnskippedEntry.TargetID == targetID, nil
 }
+
+func LogRSLEntries(ctx context.Context, repo *Repository) ([]*rsl.ReferenceEntry, map[plumbing.Hash][]*rsl.AnnotationEntry, [][]*policy.DelegationWithDepth, error) {
+	firstEntry, _, err := rsl.GetFirstEntry(repo.r)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	lastEntry, _, err := rsl.GetLatestReferenceEntryForRef(repo.r, policy.PolicyRef)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	entries, annotationMap, err := rsl.GetReferenceEntriesInRangeForRef(repo.r, firstEntry.ID, lastEntry.ID, policy.PolicyRef)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+	rules := [][]*policy.DelegationWithDepth{}
+
+	for _, entry := range entries {
```

Comment:
I could go either way on showing the policy state. We could instead add a parameter to `list-rules` to specify the state that is obtained from `rsl log` instead of trying to do everything everywhere. Basically, let's start simpler.

Reply:
I have remove outputting the current rule state for each RSL entry

Reply:
I can make another PR to add a flag to list-rules to list rules for all RSL entries